detox/src/index.js

```diff
@@ -4,7 +4,12 @@ let detox;
 
 async function init(config, params) {
   detox = new Detox(config);
-  await detox.init(params);
+  try {
+    await detox.init(params);
+  } catch (e) {
```

Comment:
I don't think this is a good idea to duplicate this particular functionality, because we already do have one:

https://github.com/wix/Detox/blob/32dec10754f2e350a0ac95d64ed49edf8c210a83/detox/src/DetoxExportWrapper.js#L83-L91